staging/cse/windows/azurecnifunc.ps1

```diff
@@ -64,6 +64,9 @@ function Set-AzureCNIConfig
 
         # $configJson.plugins[0].AdditionalArgs[0] is OutboundNAT. Replace OutBoundNAT with LoopbackDSR for IMDS.
         $configJson.plugins[0].AdditionalArgs[0] = $jsonContent
+
+        # Update RegKey to disable the incompatible HNSControlFlag. TODO: Remove it after Windows OS fixes the issue.
+        New-ItemProperty HKLM:\SYSTEM\CurrentControlSet\Services\hns\State -Name HNSControlFlag -PropertyType DWORD -Value 0

```

Comment:
Is it expected to disable the hns fix in ws2019? https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/configure-windows-vhd.ps1#L400